patientsearch/config.py

```diff
@@ -26,6 +26,31 @@ def load_json_config(potential_json_string):
     "Your account is not authorized for access, please contact an administrator",
 )
 
+DASHBOARD_COLUMNS = os.getenv(
+    "DASHBOARD_COLUMNS",
+    [
+        {
+            "label": "First Name",
+            "expr": "$.name[0].given[0]",
+        },
+        {
+            "label": "Last Name",
+            "expr": "$.name[0].family",
+        },
+        {
+            "label": "Birth Date",
+            "expr": "$.birthDate",
+        },
+        {
+            "label": "Last Accessed",
+            "defaultSort": "desc",
+            "expr": "$.meta.lastUpdated",
+            "dataType": "date",
+        },
+    ],
+)
```

Comment:
Since the result of `os.getenv()` is a string, I think we need call `json.loads()` on it, and possibly switch the fallback value to a string